cirq-google/cirq_google/api/v1/params.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
How come none of the cirq_google files register as moves?  Can we put in git hints or however it works in order for it to register as renames instead of deletes and copies?

Reply:
If you view just the final commit (`extracted cirq_google.`) these all show up properly as moves.